app/controllers/api/v1/sessions_controller.rb

```diff
@@ -0,0 +1,25 @@
+module Api
+  module V1
+    class SessionsController < DeviseTokenAuth::SessionsController
+      before_action :set_user_by_token, only: [:destroy]
```

Comment:
Try to remove these actions since you are already have them defined on parent controller